src/Main.java

```diff
@@ -0,0 +1,99 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args){
+
+         //HomeTask 1
```

Comment:
please, always add prompts for command line, like "enter the size of array", so user will know what's expected